packages/slate-react/src/components/editable.tsx

```diff
@@ -473,16 +492,20 @@ export const Editable = (props: EditableProps) => {
 
   const decorations = decorate([editor, []])
 
+  const renderedPlaceholder =
+    renderPlaceholder?.(defaultPlaceholderProps) ||
+    (placeholder && <span {...defaultPlaceholderProps}>{placeholder}</span>)
```

Comment:
Can you pull this logic out into a `defaultRenderPlaceholder` function, so that it always exists and we can simplify this to always calling the function here?